src/main/java/com/r307/arbitrader/service/NotificationServiceImpl.java

```diff
@@ -76,23 +76,27 @@ public void sendEmailNotificationBodyForEntryTrade(Spread spread, BigDecimal exi
                                                            BigDecimal longLimitPrice, BigDecimal shortVolume,
                                                            BigDecimal shortLimitPrice) {
 
-        final String longEntryString = String.format("Long entry: %s %s %s @ %s (%s slip) = %s%s\n",
+        final String longEntryString = String.format("Long entry: %s %s %s @ %s (%s slip) = %s%s (slipped from %s%s)\n",
```

Comment:
Is it important to have the difference _and_ the original amount in the message or would it be sufficient to just replace the difference with the original value?

Reply:
I think it would be useful to have both original values (price and volume*price) as the original values are the ones used by the bot to decide to trade or not.

Reply:
Sure, I was just thinking we have some redundant information now:

`1 @ 10 (2 slip) = 10 (slipped from 8)` <-- maybe we don't need the difference (2) _and_ the computed value (8)

To me, your way is easier to read than having to do the arithmetic in my head. Maybe we just say:

`1 @ 10 = 10 (slipped from 8)`